wordpress-mobile/WordPress-Android
# Feature/14706 epilogue screen for blogging reminders

Fixes #14706 

This builds off this [work in progress](https://github.com/wordpress-mobile/WordPress-Android/pull/14753) by @planarvoid.

Only need to review this [commit](https://github.com/wordpress-mobile/WordPress-Android/pull/14768/commits/2f2cc26d4b87080cada8769f4b29a44800e2741c)  here

<img width=320 src=https://user-images.githubusercontent.com/990349/122340022-1aabd180-cf85-11eb-8641-e5bfd7f189c2.png>

To test:

- Turn on the blogging reminders flag (Me/App settings/Test feature configuration - BloggingRemindersFeatureConfig)
- Create a new post
- Notice the blogging reminders prologue screen is shown
- Click on "Set goals"
- Select days screen is shown, select days, then "Notify me" button is enabled
- Click on "Notify me" button
- Notice the epilogue screen is shown.  Done button is enabled, Clicking Done button dismisses bottomsheet

Alternatively, you can get to it from

- My Site > Settings > Blogging goals then selecting days on bottom sheet


## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
NA

3. What automated tests I added (or what prevented me from doing so)
Unit tests

PR submission checklist:
